testdata/terraform/cloudflare_ruleset_zone.tf

```diff
@@ -0,0 +1,154 @@
+resource "cloudflare_ruleset" "terraform_managed_resource" {
+  kind = "zone"
+  name = "Zone sanitize ruleset"
+  phase = "http_request_sanitize"
+  zone_id = "0da42c8d2132a9ddaf714f9e7c920711"
+  rules {
+    action = "execute"
+    enabled = true
+    expression = "true"
+    id = "0789dc4343054d1e981f8c44bedc6fbd"
+    ref = "0789dc4343054d1e981f8c44bedc6fbd"
+    version = "1"
+    action_parameters {
+      overrides {
+        rules {
+          enabled = true
+          id = "78723a9e0c7c4c6dbec5684cb766231d"
+        }
+      }
+      id = "70339d97bdb34195bbf054b1ebe81f76"
+    }
+  }
+}
+
+resource "cloudflare_ruleset" "terraform_managed_resource" {
```

Comment:
this is actually an incorrect generation however it's because we don't yet support HTTP rate limits in terraform. we'll leave it for now as this will automatically get support in the future and fail the test suite on it's own to be addressed